platforms/platform_if/rtl/platform_shims/README

```diff
@@ -0,0 +1,15 @@
+Shims here are invoked by the top-level code that manages the partial reconfiguration
+boundary.  Instead of directly instantiating an AFU (e.g. ccip_std_afu), these
+intermediate shims may be mapped first.  The shims transform the interface and then
+connect to the AFU.
+
+Shims names are defined in the JSON field "platform-shim-module-name" in
+AFU top-level interface descriptions: ../../../afu_top_ifc_db/
+
+Shims typically offer automatic clock crossing and automatic platform-specific
+register stage insertion to aid in timing closure.  See the various instances of
+"clock" and "num-auto-timing-reg-stages" in ../../../platform_db/platform_defaults.json.
+
+Some modules here are sub-shims, instantiated by larger shims.  For example,
```

Comment:
I dont expect anyone to work directly with sub-shims. Just a comment.

Reply:
That's true.